### PR TITLE
7099 debt terminology

### PR DESCRIPTION
**Pull request by turadg**
refs: #7099 

## Description

Vaults and Stakes have debts but the term "loan" is misleading. This replaces it it with "vault", "debt" and other more explicit terminology.

### Security Considerations

n/a

### Scaling Considerations

n/a

### Documentation Considerations

This updates documentation. Some external documentation may need updating too. cc @Tyrosine22 

### Testing Considerations

CI

Breaking change for dapp-inter @samsiegart

### CODE REVIEW

File: packages/inter-protocol/src/stakeFactory/stakeFactoryKit.js [hunk 1]
```diff
@@ -91,8 +91,8 @@ const initState = (zcf, startSeat, manager) => {
       runWanted,
     );
     !AmountMath.isEmpty(fee) ||
-      Fail`loan requested (${runWanted}) is too small; cannot accrue interest`;
-    AmountMath.isEqual(newDebt, toMint) || Fail`loan fee mismatch`;
+      Fail`debt requested (${runWanted}) is too small; cannot accrue interest`;
+    AmountMath.isEqual(newDebt, toMint) || Fail`debt fee mismatch`;
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/src/stakeFactory/stakeFactoryKit.js:95]: The fee is a `mintFee`. Wouldn't it be better to use that terminology?

File: packages/inter-protocol/test/attestation/exampleLoanUsage.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 /**
- * ** Create loan:
+ * ** Create vault:
  *
  * requires an NFT attestation of brand BldAttLoc, and the amount of
- * RUN that can be loaned is about 10% of the value of BLD represented
+ * RUN that can be vaulted is about 10% of the value of BLD represented
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/test/attestation/exampleLoanUsage.js:5]: no way.

'lent', perhaps.  The README says the protocol 'lends' on the basis of collateral.  I think 'loaned' is acceptable.

You could also say that the amount that can be 'withdrawn' is limited.

File: packages/inter-protocol/test/vaultFactory/test-vault-collateralization.js [hunk 3]
```diff
@@ -43,15 +43,15 @@ test('add debt to vault under LiquidationMarging + LiquidationPadding', async t
...
+  // we can borrow and pay it back
   await vd.giveCollateral(100n, aeth, MARGIN_HOP);
   await vd.giveMinted(MARGIN_HOP, aeth, 100n);
 
   // but once we bump LiquidationPadding they are under
   await md.setGovernedParam('LiquidationPadding', run.makeRatio(20n, 1n), {
     key: { collateralBrand: aeth.brand },
   });
-  // ...so we can't take the same loan out
+  // ...so we can't take the same debtout
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/test/vaultFactory/test-vault-collateralization.js:54]:
```suggestion
  // ...so we can't take the same debt out
```

File: packages/inter-protocol/test/vaultFactory/test-vaultFactory.js [hunk 4]
```diff
@@ -477,32 +473,32 @@ test.skip('price drop', async t => {
     reserveKit: { reserveCreatorFacet },
   } = services;
 
-  // Create a loan for 270 Minted with 400 aeth collateral
+  // Create a vault for 270 Minted with 400 aeth collateral
   const collateralAmount = aeth.make(400n);
-  const loanAmount = run.make(270n);
+  const debt = run.make(270n);
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/test/vaultFactory/test-vaultFactory.js:478]: `borrowedAmount` would be better. `debt` includes the fee.

File: packages/inter-protocol/test/vaultFactory/test-vaultFactory.js [hunk 5]
```diff
@@ -600,16 +596,16 @@ test.skip('price drop', async t => {
...
-  t.context.loanTiming = {
+  t.context.interestTiming = {
     chargingPeriod: 2n,
     recordingPeriod: 10n,
   };
   t.context.aethInitialLiquidity = aeth.make(900n);
 
   // The borrower will deposit 4 Aeth, and ask to borrow 470 Minted. The
-  // PriceAuthority's initial quote is 180. The max loan on 4 Aeth would be 600
+  // PriceAuthority's initial quote is 180. The max vault on 4 Aeth would be 600
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/test/vaultFactory/test-vaultFactory.js:606]: vault's the wrong word here. maybe `borrowed`?

File: packages/inter-protocol/test/vaultFactory/test-vaultFactory.js [hunk 6]
```diff
@@ -628,15 +624,15 @@ test.skip('price falls precipitously', async t => {
   const { vaultFactory, lender } = services.vaultFactory;
 
   const { reserveCreatorFacet } = services.reserveKit;
-  // Create a loan for 370 Minted with 400 aeth collateral
+  // Create a vault for 370 Minted with 400 aeth collateral
   const collateralAmount = aeth.make(400n);
-  const loanAmount = run.make(370n);
+  const debt = run.make(370n);
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/test/vaultFactory/test-vaultFactory.js:629]: `borrowedAmount`

File: packages/inter-protocol/test/vaultFactory/test-vaultFactory.js [hunk 7]
```diff
@@ -1309,7 +1305,7 @@ test('adjust balances after interest charges', async t => {
 
   const { lender } = services.vaultFactory;
 
-  trace('0. Take out loan');
+  trace('0. Take out vault');
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/test/vaultFactory/test-vaultFactory.js:1308]:
```suggestion
  trace('0. Create vault');
```

File: packages/inter-protocol/test/vaultFactory/test-vaultFactory.js [hunk 8]
```diff
@@ -1488,7 +1484,7 @@ test('transfer vault', async t => {
 });
 
 // Alice will over repay her borrowed Minted. In order to make that possible,
-// Bob will also take out a loan and will give her the proceeds.
+// Bob will also take out a vault and will give her the proceeds.
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/test/vaultFactory/test-vaultFactory.js:1487]:
```suggestion
// Bob will also create a vault and will give her the proceeds.
```

**Review by Chris-Hibbert — CHANGES_REQUESTED** [review]
Lots of very local changes that will interact with #7047.  Can it wait?

File: packages/inter-protocol/test/attestation/exampleLoanUsage.js (hunk 2 above)
Comment (turadg) [reply packages/inter-protocol/test/attestation/exampleLoanUsage.js:5]: hehe this was find-replace. I'll use one of your suggestions.

**Comment by turadg** [comment]
> Lots of very local changes that will interact with #7047. Can it wait?

ok. I'll put it with https://github.com/Agoric/agoric-sdk/pull/7094 in the queue of things to hold until after 7047.

**Comment by turadg** [comment]
quite obsolete

**Pull request closed without merging**